.circleci/config.yml

```diff
@@ -95,6 +40,81 @@ defaults:
       tags:
         only: /.*/
 
+
+executors:
+  python_with_flowdb: # Synthetic data generation execution environment
+    parameters:
+      flowdb_image:
+        type: enum
+        enum: ["synthetic-data", "testdata"]
+        default: "testdata"
+      generator:
```

Comment:
Maybe call this `synthetic_data_generator` for clarity (in line with the env var `SYNTHETIC_DATA_GENERATOR`)?